src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/readytolist/ReadyToListAboutToSubmitHandler.java

```diff
@@ -25,24 +44,48 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
     }
 
     @Override
-    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback,
-                                                          String userAuthorisation) {
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback, String userAuthorisation) {
         if (!canHandle(callbackType, callback)) {
             throw new IllegalStateException("Cannot handle callback.");
         }
+
         final CaseDetails<SscsCaseData> caseDetails = callback.getCaseDetails();
-        final SscsCaseData sscsCaseData = caseDetails.getCaseData();
+        final SscsCaseData sscsCaseData = caseDetails.getCaseData(); //Maybe handle in each method instead of HEAD
 
-        PreSubmitCallbackResponse<SscsCaseData> callbackResponse = new PreSubmitCallbackResponse<>(sscsCaseData);
+        if(schedulingAndListingFeature){
+            boolean listAssist = checkIfListAssist();
+            if(listAssist){
+                //Build response to be enium value and case ID
+                JSONObject buildJson = new JSONObject();
+                buildJson.put("STATE", STATE.HEARING_CREATED);
+                buildJson.put("CaseID", sscsCaseData.getCcdCaseId());
+                PreSubmitCallbackResponse<SscsCaseData> callbackResponse = new PreSubmitCallbackResponse<>(sscsCaseData); //Placeholder
```

Comment:
You want to set the type of hearing it is on the case data. I believe there is a hearings list which you can add a new field corresponding to the type of hearing - if it is list assist, add to the queue. In Evidence share you'll need to add logic to only send to GAPS if the type of hearing is GAPS